README.md

```diff
@@ -48,13 +48,16 @@ This repository replaces `iocage_legacy`. To upgrade to the current version:
 1. Migrate the jails. This can be done by running `iocage list` as root
 1. Start the jails (`service iocage onestart`)
 
+## Links
+
+- **[iocage Project Website](https://iocage.github.io/)**
+- **[Documentation](http://iocage.readthedocs.org/en/latest/index.html)**
+- **[Mailing list](https://groups.google.com/forum/#!forum/iocage)**
```

Comment:
i'd like to point out that at least twice a week, someone shows up in #iocage on freenode, and i tell them to just open an issue

Reply:
Open an issue about what?

Reply:
about their questions / bugs